setup.sh

```diff
@@ -184,12 +197,13 @@ if [ $build_gazebo -eq 0 ] && [ $build_elektron -eq 0 ] && [ $build_orocos -eq 0
 	build_velma=1
 fi
 
+# It is okay not to install Gazebo
 ### Check if required workspaces are installed in the /opt directory
-if [[ ! -d "/opt/ws_gazebo" ]] && [ $build_gazebo -eq 0 ]; then
-	# Gazebo is required for everything else
-	printError "Gazebo is not in the /opt directory, please install it or add <-g> flag to the script."
-	exit 1
-fi
+#if [[ ! -d "/opt/ws_gazebo" ]] && [ $build_gazebo -eq 0 ]; then
```

Comment:
Jeśli jest w historii gita, to nie ma co zostawiać takich zakomentowanych bloków (zaśmieca plik)